shared/cfg/base.cfg

```diff
@@ -129,22 +129,25 @@ q35, arm64-pci, arm64-mmio:
 # Serial port support
 # You can assign more than one serial to guest with this parameter.
 # like:
-# serials = "S1 S2 S3"
-serials = "serial0"
-
-# virtio_serialport and virtio_console ports
-# virtio_ports = "vs1 vs2 vc1"
-# Port types
-# virtio_port_type = serialport
-# virtio_port_type_vc1 = console
-# Host socket type
-# virtio_port_chardev = spicevmc
-# virtio_port_chardev_vc1 = socket
-# chardev name prefix (port index will be appended)
+# serials = "serial0 vs1 vs2 vc1"
+# serial_type_vs1 = virtserialport
+# serial_type_vc1 = virtconsole
+# Chardev backend type: 'unix_socket' by default
+# chardev_backend_vs1 = unix_socket
+# chardev_backend_vc1 = tcp_socket
+# Chardev host for tcp/udp socket backend
+# chardev_host = $host_ip (127.0.0.1 by default)
+# chardev_ipv4 = yes
+# chardev_ipv6 = yes
```

Comment:
By default,  no `ipv4` and `ipv6`. It is
```
                 'ipv4': params.get('chardev_ipv4'),
                 'ipv6': params.get('chardev_ipv6')})
```
qemu command line `-chardev udp,host=127.0.0.1,id=chardev_vc2,port=5746`

Reply:
@yanan-fu It's just an example to show how to use it, not mean default here.

Reply:
@vivianQizhu  ok, for usage, `yes` `on` `True` all can use:
```
        for opt in ["server", "telnet", "nowait",
                    "ipv4", "ipv6", "nodelay"]:
            if params.get(opt) in ["yes", "on", True]:
                params[opt] = "NO_EQUAL_STRING"
            elif opt in params:
                del params[opt]
```

Reply:
@yanan-fu I want to keep it simple, so only list one usage here, to not disturb users.

Reply:
It is ok for me ~